<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_texto_oculto</class>
 <widget class="QWidget" name="frm_texto_oculto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codificação de Texto</string>
  </property>
  <widget class="QPushButton" name="bt_encriptar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 51, 59);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Codificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_desencriptar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 233, 53);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Decodificar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_texto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_limpar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 132, 228);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
